snippets/find_multiples.md

```diff
@@ -0,0 +1,17 @@
+---
+title: find_multiples
```

Comment:
I think this should be named `arithmetic_progression` instead as it would make more sense and provide more value to users.